azurerm/internal/services/trafficmanager/traffic_manager_endpoint_resource.go

```diff
@@ -116,6 +116,16 @@ func resourceArmTrafficManagerEndpoint() *pluginsdk.Resource {
 				Optional: true,
 			},
 
+			"min_child_endpoints_ipv4": {
+				Type:     pluginsdk.TypeInt,
+				Optional: true,
+			},
+
+			"min_child_endpoints_ipv6": {
```

Comment:
could we rename these too
```suggestion
			"minimum_required_child_endpoints_ipv4": {
				Type:     pluginsdk.TypeInt,
				Optional: true,
			},

			"minimum_required_child_endpoints_ipv6": {
```